package.json

```diff
@@ -4,6 +4,7 @@
   "description": "simple community management",
   "main": "doorman.js",
   "scripts": {
+    "start": "node doorman.js",
```

Comment:
Switching to using `npm start` as the recommended command for launching the bot.  Will also add section to README on running as a service (probably using `pm2`).